adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala

```diff
@@ -169,7 +170,7 @@ case class NucleotideContigFragmentRDD(
    * @param region Reference region over which to get sequence.
    * @return String of bases corresponding to reference sequence.
    */
-  def getReferenceString(region: ReferenceRegion): String = {
+  def extract(region: ReferenceRegion): String = {
```

Comment:
Wouldn't this read so much better if this class was a `SequenceRDD` (or `SliceRDD`, depending on how the data were loaded) and this method was
```scala
def slice(region: ReferenceRegion): Slice = { ... }
```
See https://github.com/bigdatagenomics/bdg-formats/pull/83